android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -68,6 +68,7 @@ data class QuestionnaireConfig(
   val linkIds: List<LinkIdConfig>? = null,
   val showSubmitAnywayButton: String = "false",
   val showSubmissionConfirmationDialog: String = "false",
+  val repeatGroup: RepeatGroupConfig? = null,
```

Comment:
Would it be possible to rename this?

Reply:
PS. Can we use a JSON array for `initialResourceIdslinkId`, then handle the processing of the list item via code? It's unclear that the property requires a CSV just by reading.

Reply:
From our conversation, we resolved to use [QuestionnaireConfig.linkId](https://github.com/opensrp/fhircore/blob/8133797e9d34feb33ce6a2dda9da9c0ce309fe5c/android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt#L68) instead of these new configs. A new LinkeIdType will be provided for this particular use case. I'm gonna request the change so that this PR will only be merged once the refactor has been applied.